src/lib.rs

```diff
@@ -70,35 +70,56 @@
 //!
 //! ```{rust}
 //! use semver::Version;
+//! use std::error::Error;
```

Comment:
Ah, sorry I didn't pick this up before! We should probably hide these `use std::error::Error` lines too since they don't actually relate to the example.

I would probably just remove that import altogether and change the function signatures to:

```rust
# fn try_increment_patch() -> Result<(), Box<::std::error::Error>> {
```